cmd/contour/shutdownmanager.go

```diff
@@ -136,13 +139,16 @@ func (s *shutdownmanagerContext) shutdownReadyHandler(w http.ResponseWriter, r *
 // shutdownHandler is called from a pod preStop hook, where it will block pod shutdown
 // until envoy is able to drain connections to below the min-open threshold.
 func (s *shutdownContext) shutdownHandler() {
+
+	fmt.Println("---admin-port: ", s.adminPort)
```

Comment:
nit: Remove the log.

Reply:
Hi @stevesloka, I removed the log and for that minimal change the codecoverage tool blocks my PR. With the log line (1 line more), the PR is allowed. I don't understand why. What would you suggest to get the work done? I'm not sure how to test those functions in which I made the little adds, because I didn't find any test on those too.